### PR TITLE
Remove unused environment variable

### PR DESCRIPTION
**Pull request by david-mears-dfe**
We use `ALGOLIA_WRITE_API_KEY` everywhere. It has the same permissions as `ALGOLIA_SEARCH_API_KEY` and then some.

### CODE REVIEW

File: terraform/app/container-definitions/web_container_definition.json [hunk 1]
```diff
@@ -209,10 +209,6 @@
       {
         "name": "ALGOLIA_WRITE_API_KEY",
         "value": "${algolia_write_api_key}"
-      },
```
Comment (cesidio) [inline terraform/app/container-definitions/web_container_definition.json:212]: We probably do not need this file anymore @saliceti

Comment (saliceti) [reply terraform/app/container-definitions/web_container_definition.json:212]: @david-mears-dfe can you remove the whole `terraform/container-definitions` directory please?

**Comment by cpjmcquillan** [comment]
Will #1721 necessitate this API key still exists? Client search needs far fewer permissions and the search API key can be exposed in the client safely

**Comment by david-mears-dfe** [comment]
> Will #1721 necessitate this API key still exists? Client search needs far fewer permissions and the search API key can be exposed in the client safely

Yes.

**Pull request closed without merging**